codespell_lib/data/dictionary.txt

```diff
@@ -16917,7 +16928,8 @@ medeival->medieval
 medevial->medieval
 medhod->method
 medhods->methods
-mediciney->mediciny
+mediciney->medicine
+mediciny->medicine
```

Comment:
Perhaps suggest medicinal too "having healing properties" so it's medicine-like and perhaps what they meant.

Reply:
done